### PR TITLE
feat(quantity): remove component fixed size

### PR DESCRIPTION
**Pull request by adrienbusin**
## Changes description 🧑‍💻

 remove component fixed size
## Context 🤔

https://github.com/Decathlon/vitamin-design/issues/162
## Checklist ✅

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Please, don't request directly from your main!
- [x] Check commits & PR names matches our requested structure. It must follow the https://www.conventionalcommits.org pattern.
- [x] Check compose contract convention. It must follow conventions described [here](/CONVENTIONS.md).
- [x] Check your code additions will fail neither code linting checks.
- [x] I have reviewed the submitted code.
- [x] I have tested on a phone device/emulator.
- [ ] I have tested on a tablet device/emulator.
- [ ] I have tested on a large screen device/emulator.
- [x] If it includes design changes, please ask for a review `design-system-core-team-design` GitHub team.

### CODE REVIEW

**Comment by ManonPolle** [comment]
Waiting for this issue to be closed : https://github.com/Decathlon/vitamin-design/issues/162

**Comment by dktbitrise** [comment]
### You can download the apk here :

**Comment by dktbitrise** [comment]
### You can download the apk here :

**Review by ManonPolle — APPROVED** [review]

File: quantity-pickers/src/main/java/com/decathlon/vitamin/compose/quantity/pickers/VitaminQuantityPickers.kt
Comment (ManonPolle) [inline quantity-pickers/src/main/java/com/decathlon/vitamin/compose/quantity/pickers/VitaminQuantityPickers.kt:68]: We may need to add min width after design approval. PR is blocked until that

**Review by ManonPolle — CHANGES_REQUESTED** [review]

File: quantity-pickers/src/main/java/com/decathlon/vitamin/compose/quantity/pickers/VitaminQuantityPickers.kt
Comment (adrienbusin) [reply quantity-pickers/src/main/java/com/decathlon/vitamin/compose/quantity/pickers/VitaminQuantityPickers.kt]: done with : 
.widthIn(min = 154.dp)

**Comment by dktbitrise** [comment]
### You can download the apk here :
**sample-debug.apk :** 
https://app.bitrise.io/app/e9ae480151be4514/build/971d325c-80fc-4dd2-bd72-fe1d3bb3bb9b/artifact/acced12277893bb2/p/3b4a28b781484fbea2de37edc55b482b

**Review by ManonPolle — APPROVED** [review]

**Comment by dktbitrise** [comment]
### You can download the apk here :
**sample-debug.apk :** 
https://app.bitrise.io/app/e9ae480151be4514/build/474765d0-9c84-4976-89f0-2e3059893254/artifact/a40ab25a568a09be/p/0680e566a02847085483d996b60167d8

**Review by SimonLeclercq — APPROVED** [review]

**Pull request merged**
